custom/icds/location_reassignment/download.py

```diff
@@ -201,78 +202,105 @@ def _add_extra_sheets(workbook):
             worksheet.append(headers)
 
 
-class Households(object):
+class DownloadCases(metaclass=ABCMeta):
```

Comment:
extracted an abstract class to declare/method names that classes for downloading cases should implement.